src/flow.js

```diff
@@ -12,7 +13,8 @@ async function flow() {
 
 
   // this will be set by the user later. setting it manually for now
-  const root = './';
+  const root = PATH.resolve(__dirname, './');
```

Comment:
@digitalgnome I'm really curious about this part - does it get messed up with just './'? I would imagine the way you're doing it here is much more stable but I'd love to know if you saw any weird issues with the old version so I understand the flaws of it better.